assets/testrail.rb

```diff
@@ -182,6 +182,36 @@ def get_suite_stats_for_project_by_suite(project_id, suite_id)
 		  return [total_cases_count, possible_automation_pct, automation_coverage_pct]
 		end
 
+
+
+		def get_suite_stats(suite_id)
+			suite = @client.send_get('get_suite/%s' % suite_id)
+			test_cases = @client.send_get("get_cases/#{suite['project_id']}&suite_id=#{suite_id}")
+
+			total_cases_count = test_cases.size
+			untriaged_cases = 0
+			suitable_cases = 0
+			unsuitable_cases = 0
+			completed_cases = 0
+
+			for case_ in test_cases
+				case case_['custom_automation_status']
+					when 1
+						untriaged_cases+=1
+					when 2
+						suitable_cases+=1
+					when 3
+						unsuitable_cases+=1
+					when 4
+						completed_cases+=1
+				end
+			end
+
+
+			return [total_cases_count, untriaged_cases, suitable_cases, unsuitable_cases, completed_cases]
```

Comment:
rather than returning this as an array, it would be good to use a hash so you can refer to each index by name in each job. something like ```return {:total_cases => total_cases_count, :untriaged_cases =>...}```

and then in the job ```untriaged_tasks_count = test_cases_counts[:untriaged_cases]``` rather than the array index so it's easier to read